### ISSUE TITLE
AutoEncoder

### ISSUE THREAD

**Issue by IanQS** [IanQS/krak_trader#2]
Move to Keras for architecture to see if you're fucking anything up. Create a Keras and TF version to test improvements

- [X] Keras low error
- [x] Validation set low error

After this, make a new PR for convolutional arch
